components/KebabPopUp.tsx

```diff
@@ -16,7 +16,7 @@ export const KebabPopUp: React.FC<KebabPopUpProps> = (props) => {
   const controller = useKebabPopUp(props);
   const { t } = useTranslation('HomeScreenKebabPopUp');
   return (
-    <Column>
+    <Column testID={props.testID}>
```

Comment:
List item in overlay should have testID, that will only help in automation.
In automation, we have to click on items say pin or unpin, view activity logs so these items should have testIDs